com.gitlab.newsflash.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "com.gitlab.newsflash",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "com.gitlab.newsflash",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications",
+        "--filesystem=xdg-download"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
Network access is forbidden during build time, you should be able to include all the dependencies if you run `ninja dist`

Reply:
How would I do that?
I though all I tell flathub is `"buildsystem" : "meson",` and it will figure out what commands to run.

Reply:
Yes, but the tarball doesn't include all the requires crates to build it. You would need to create the tarball yourself by doing `meson _builddir; ninja -C _builddir dist` and use that tarball for building the application

Reply:
okay, should be good now
f1a358e509e1f5969744bad5e6e794aea94eb070